examples/tutorial.rs

```diff
@@ -221,8 +221,7 @@ fn example1() -> Result<(), Box<dyn Error>> {
     let start = Instant::now();
     let cookies = names
         .iter()
-        .map(|name| conn.intern_atom(false, name.as_bytes()))
-        .collect::<Vec<_>>();
+        .map(|name| conn.intern_atom(false, name.as_bytes()));
     for (i, atom) in cookies.into_iter().enumerate() {
```

Comment:
Well... no. This defeats the point of this example (which is to send all the `InternAtom` requests before starting to wait for their replies). Now, we are back to "send one request and wait for its reply". Hopefully this difference shows up when running the example and I am not just imagining it...

Reply:
Yep, you're right, I didn't realize that the `collect` was to actually execute the closure passed to `map`.